deploy/sre-prometheus/extended-logging/config.yaml

```diff
@@ -0,0 +1,9 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  # if we ever remove this, do not remove the resources
```

Comment:
Why?

Reply:
Not sure off the top of my head.
Will TAL later and reply